docs/content/commands/rclone_move.md

```diff
@@ -26,6 +26,8 @@ move will be used, otherwise it will copy it (server side if possible)
 into `dest:path` then delete the original (if no errors on copy) in
 `source:path`.
 
+If you want to delete empty source directories after move, use the --delete-empty-source-dirs flag.
```

Comment:
These docs get added automatically by `make commanddocs` which is run as part of making the release, so you don't need to update them here.  It isn't a problem though!

Reply:
Good to know this! I've gone through the make file just in case I've missed anything.